### PR TITLE
EREGCSC-2008 -- Admin login update

### PR DESCRIPTION
**Pull request by thwalker6**
Resolves # EREGCSC 2008

**Description-**

Adds an error message to the login form and a link to EUA authentication.  Authention URL is just google for now until we get a proper set up.

**This pull request changes...**

- Adds some preliminary commented out stuff for okta in settins.py
- A new error message will pop up at the bottom on failed login. This is a place holder.
- A line for a link to login to EUA.  Also a placeholder.
- More relevant headings in admin panel for our admin.
- 
**Steps to manually verify this change...**

1. Go to the admin panel.
2. You will see a link for logging into EUA.  The link is currently just google.
3. Logging in incorrectly will display an extra error message.  Will change when EUA is set up.
4. Log in successfully.  It wont just say django admin panel but stuff relevant to eregs.

### CODE REVIEW

File: solution/backend/cmcs_regulations/urls.py [hunk 1]
```diff
@@ -31,9 +31,13 @@
...
+    path('admin/login/', auth_views.LoginView.as_view(template_name='admin/login.html'), name='login'),
     path('admin/', admin.site.urls, name="admin"),
     path('robots.txt', TemplateView.as_view(template_name="robots.txt", content_type="text/plain")),
     path('__debug__/', include('debug_toolbar.urls')),
     path('sitemap.xml', sitemap, {'sitemaps': sitemaps}, name='django.contrib.sitemaps.views.sitemap'),
     path('latest/feed/', ResourceFeed()),
 ]
+admin.site.site_header = "e-Regs"
+admin.site.site_title = 'e-Regs Admin Panel'
+admin.site.index_title = 'Medicaid & CHIP eRgulations Admin Panel'
```
Comment (cgodwin1) [inline solution/backend/cmcs_regulations/urls.py:43]: eRgulations -> eRegulations

File: solution/backend/cmcs_regulations/urls.py [hunk 2]
```diff
@@ -31,9 +31,13 @@
...
     path('favicon.ico', RedirectView.as_view(url=settings.STATIC_URL + 'images/favicon/favicon.ico')),
+    path('admin/login/', auth_views.LoginView.as_view(template_name='admin/login.html'), name='login'),
     path('admin/', admin.site.urls, name="admin"),
     path('robots.txt', TemplateView.as_view(template_name="robots.txt", content_type="text/plain")),
     path('__debug__/', include('debug_toolbar.urls')),
     path('sitemap.xml', sitemap, {'sitemaps': sitemaps}, name='django.contrib.sitemaps.views.sitemap'),
     path('latest/feed/', ResourceFeed()),
 ]
+admin.site.site_header = "e-Regs"
+admin.site.site_title = 'e-Regs Admin Panel'
```
Comment (cgodwin1) [inline solution/backend/cmcs_regulations/urls.py:42]: e-Regs -> eRegs

File: solution/backend/cmcs_regulations/urls.py [hunk 3]
```diff
@@ -31,9 +31,13 @@
...
     path('', include('regulations.urls')),
     path('favicon.ico', RedirectView.as_view(url=settings.STATIC_URL + 'images/favicon/favicon.ico')),
+    path('admin/login/', auth_views.LoginView.as_view(template_name='admin/login.html'), name='login'),
     path('admin/', admin.site.urls, name="admin"),
     path('robots.txt', TemplateView.as_view(template_name="robots.txt", content_type="text/plain")),
     path('__debug__/', include('debug_toolbar.urls')),
     path('sitemap.xml', sitemap, {'sitemaps': sitemaps}, name='django.contrib.sitemaps.views.sitemap'),
     path('latest/feed/', ResourceFeed()),
 ]
+admin.site.site_header = "e-Regs"
```
Comment (cgodwin1) [inline solution/backend/cmcs_regulations/urls.py:41]: e-Regs -> eRegs

File: solution/backend/cmcs_regulations/templates/admin/login.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                justify-content: center;
+            }
+        </style>
+    {% endblock %}
+   {{ block.super }}
+   <div class="okta_login_container">
+       Or Login With&nbsp; <a href="https://www.google.com/">EUA</a>
+   </div>
+   {% if form.errors %}
+    <p>Invalid credentials.  Please contact an EREGS administrator if you need access.</p>
```
Comment (cgodwin1) [inline solution/backend/cmcs_regulations/templates/admin/login.html:26]: EREGS -> eRegs

File: solution/backend/cmcs_regulations/templates/admin/login.html [hunk 5]
```diff
@@ -0,0 +1,28 @@
+{% extends "admin/login.html" %}
+
+{% block title %}E-Regulations Admin Login{% endblock %}
```
Comment (cgodwin1) [inline solution/backend/cmcs_regulations/templates/admin/login.html:3]: E-Regulations -> eRegulations

**Review by cgodwin1 — CHANGES_REQUESTED** [review]
Minor spelling adjustments, otherwise good

File: solution/backend/cmcs_regulations/templates/admin/login.html (hunk 5 above)
Comment (thwalker6) [reply solution/backend/cmcs_regulations/templates/admin/login.html:3]: updated

File: solution/backend/cmcs_regulations/templates/admin/login.html (hunk 4 above)
Comment (thwalker6) [reply solution/backend/cmcs_regulations/templates/admin/login.html:26]: updated

**Review by cgodwin1 — APPROVED** [review]

**Pull request merged**